src/celeritas/inp/SurfacePhysics.hh

```diff
@@ -0,0 +1,206 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/inp/SurfacePhysics.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <algorithm>
+#include <map>
+
+#include "corecel/Types.hh"
+#include "corecel/cont/Range.hh"
+#include "corecel/math/SoftEqual.hh"
+#include "geocel/Types.hh"
+#include "celeritas/inp/Grid.hh"
+
+namespace celeritas
+{
+namespace inp
+{
+//---------------------------------------------------------------------------//
+// SURFACE DESCRIPTION: Reflectivity and models for surface normals.
+//---------------------------------------------------------------------------//
+/*!
+ * Surface reflectivity index, which can be a user-defined Grid, which is a
+ * function of energy (wavelength), or constant.
+ */
+struct ReflectionGrid
+{
+    Grid grid;
```

Comment:
```suggestion
    Grid reflectivity;
```

Reply:
This and the previous comment made me realize the nomenclature inconsistency. How about this instead, since both grid/analytic are reflectivity models:

```cpp
struct ReflectivityModels
{
    std::map<SurfaceLayer, GridReflection> grid;
    std::map<SurfaceLayer, FresnelReflection> fresnel;
};
```